source/renderer/app/components/staking/DelegationInfo.scss

```diff
@@ -0,0 +1,29 @@
+@import './stakingConfig';
+
+.component {
+  margin: 20px auto;
+  text-align: center;
+  width: 800px;
```

Comment:
@yakovkaravelov this won't work. We should never define fixed widths as it breaks the responsive behavior. Please remove this and use left and right paddings to prevent content sticking to the borders of the wrapper element.

- The content is currently cut-off on small screens:
![Screen Shot 2019-05-17 at 10 43 31](https://user-images.githubusercontent.com/376611/57915522-44b7d480-7891-11e9-940d-74a1653dc395.png)

- The content is not expanding to the available width on large screens:
![Screen Shot 2019-05-17 at 10 43 43](https://user-images.githubusercontent.com/376611/57915535-4d100f80-7891-11e9-9449-984330e100dd.png)